redis.conf

```diff
@@ -1443,6 +1443,12 @@ lua-time-limit 5000
 #
 # cluster-config-file nodes-6379.conf
 
+# Enabling custom hashing allows any user to their own hashing function which
+# can be defined in the custom_hash.c file. By default the custom hashing mode
+# is disabled.
```

Comment:
Such text should make it clear that it's the cluster slot hashing we're talking about. "Hash function" can also refer to the hash function in dict.